app.py

```diff
@@ -76,25 +82,26 @@ class ValidatorInfo(BaseModel):
     all_uid_info: dict = {}
     sha: str = ""
 
-
 class ImageGenerationService:
     def __init__(self):
         self.subtensor = bt.subtensor("finney")
         self.metagraph = self.subtensor.metagraph(23)
+        # mongoDBConnectUri = f"mongodb://{MONGO_DB_USERNAME}:{MONGO_DB_PASSWORD}@localhost:27017"
```

Comment:
How about the case mongodb is server by other VM, Atlas?

Reply:
Yeah, in that case, we need to use the commented Uri as the mongoDBConnectUri. I commented it to test on my local environment.